dadguide/monster_graph.py

```diff
@@ -41,11 +49,13 @@ def build_graph(self):
             mtoawo[a.monster_id].append(a)
 
         for m in ms:
-            self.graph.add_node(m.monster_id,
-                                awakenings=mtoawo[m.monster_id],
-                                leader_skill=lss.get(m.leader_skill_id),
-                                active_skill=ass.get(m.active_skill_id),
-                                series=ss.get(m.series_id))
+            m_model = MonsterModel(awakenings=mtoawo[m.monster_id],
+                                   leader_skill=lss.get(m.leader_skill_id),
+                                   active_skill=ass.get(m.active_skill_id),
+                                   series=ss.get(m.series_id),
+                                   is_farmable=m.drop_id is not None
+                                   )
```

Comment:
nit: formatting. put ) on previous line, or put newline between `MonsterModel(` and `awakenings=`

Reply:
added newline